requirements.txt

```diff
@@ -0,0 +1,5 @@
+certifi==2017.7.27.1
```

Comment:
Why is this file needed? Dependencies are defined here: https://github.com/ButterCMS/buttercms-python/blob/master/setup.py#L22